dense_vector/index.json

```diff
@@ -0,0 +1,28 @@
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 2,
+      "number_of_replicas": 0,
+      "refresh_interval": -1
```

Comment:
I am wondering, why do we need this as we aren't doing concurrent indexing and querying? My understanding is that starting with 7.3.0 there's no refresh if indices haven't received a search request within the last 30s, so there shouldn't be any refreshing in the bulk indexing task.

Reply:
Sorry about that, I did not mean to commit this. I reverted the change.